Description: POD markup corrections
The bulk of the corrections are an extra '=cut' at the end of files
which cause 'pod2man' to include a section at the end of the man page
labelled 'POD ERRORS'.
.
The exceptions were:
.
lib/Locale/Messages.pm had an '=item' that was not enclosed in '=over'
and '=back' clauses.
.
lib/Locale/TextDomain.pm has a '=back' clause without a prior blank line.
Bug-Debian: htps://bugs.debian.org/945988
Author: Ken Ibbotson <keni@computer.org>
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=134035
Bug: https://rt.cpan.org/Ticket/Display.html?id=134035
Last-Update: 2021-01-11

--- a/lib/Locale/Messages.pm
+++ b/lib/Locale/Messages.pm
@@ -995,8 +995,12 @@
 
 =head1 OTHER EXPORTS
 
+=over 4
+
 =item B<select_package PACKAGE>
 
+=back
+
 =head1 USAGE
 
 A complete example:
--- a/lib/Locale/Recode.pm
+++ b/lib/Locale/Recode.pm
@@ -362,4 +362,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/Recode/_Aliases.pm
+++ b/lib/Locale/Recode/_Aliases.pm
@@ -741,4 +741,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/Recode/_Conversions.pm
+++ b/lib/Locale/Recode/_Conversions.pm
@@ -454,4 +454,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData.pm
+++ b/lib/Locale/RecodeData.pm
@@ -171,4 +171,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ASMO_449.pm
+++ b/lib/Locale/RecodeData/ASMO_449.pm
@@ -867,4 +867,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ATARI_ST.pm
+++ b/lib/Locale/RecodeData/ATARI_ST.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ATARI_ST_EURO.pm
+++ b/lib/Locale/RecodeData/ATARI_ST_EURO.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP10007.pm
+++ b/lib/Locale/RecodeData/CP10007.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1250.pm
+++ b/lib/Locale/RecodeData/CP1250.pm
@@ -1128,4 +1128,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1251.pm
+++ b/lib/Locale/RecodeData/CP1251.pm
@@ -1136,4 +1136,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1252.pm
+++ b/lib/Locale/RecodeData/CP1252.pm
@@ -1128,4 +1128,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1253.pm
+++ b/lib/Locale/RecodeData/CP1253.pm
@@ -1104,4 +1104,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1254.pm
+++ b/lib/Locale/RecodeData/CP1254.pm
@@ -1124,4 +1124,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1256.pm
+++ b/lib/Locale/RecodeData/CP1256.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CP1257.pm
+++ b/lib/Locale/RecodeData/CP1257.pm
@@ -1113,4 +1113,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CSN_369103.pm
+++ b/lib/Locale/RecodeData/CSN_369103.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/CWI.pm
+++ b/lib/Locale/RecodeData/CWI.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/DEC_MCS.pm
+++ b/lib/Locale/RecodeData/DEC_MCS.pm
@@ -1107,4 +1107,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_AT_DE.pm
+++ b/lib/Locale/RecodeData/EBCDIC_AT_DE.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_AT_DE_A.pm
+++ b/lib/Locale/RecodeData/EBCDIC_AT_DE_A.pm
@@ -934,4 +934,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_CA_FR.pm
+++ b/lib/Locale/RecodeData/EBCDIC_CA_FR.pm
@@ -986,4 +986,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_DK_NO.pm
+++ b/lib/Locale/RecodeData/EBCDIC_DK_NO.pm
@@ -945,4 +945,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_DK_NO_A.pm
+++ b/lib/Locale/RecodeData/EBCDIC_DK_NO_A.pm
@@ -934,4 +934,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_ES.pm
+++ b/lib/Locale/RecodeData/EBCDIC_ES.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_ES_A.pm
+++ b/lib/Locale/RecodeData/EBCDIC_ES_A.pm
@@ -934,4 +934,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_ES_S.pm
+++ b/lib/Locale/RecodeData/EBCDIC_ES_S.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_FI_SE.pm
+++ b/lib/Locale/RecodeData/EBCDIC_FI_SE.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_FI_SE_A.pm
+++ b/lib/Locale/RecodeData/EBCDIC_FI_SE_A.pm
@@ -934,4 +934,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_FR.pm
+++ b/lib/Locale/RecodeData/EBCDIC_FR.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_IS_FRISS.pm
+++ b/lib/Locale/RecodeData/EBCDIC_IS_FRISS.pm
@@ -915,4 +915,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_IT.pm
+++ b/lib/Locale/RecodeData/EBCDIC_IT.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_PT.pm
+++ b/lib/Locale/RecodeData/EBCDIC_PT.pm
@@ -934,4 +934,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_UK.pm
+++ b/lib/Locale/RecodeData/EBCDIC_UK.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/EBCDIC_US.pm
+++ b/lib/Locale/RecodeData/EBCDIC_US.pm
@@ -944,4 +944,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ECMA_CYRILLIC.pm
+++ b/lib/Locale/RecodeData/ECMA_CYRILLIC.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/GEORGIAN_ACADEMY.pm
+++ b/lib/Locale/RecodeData/GEORGIAN_ACADEMY.pm
@@ -1126,4 +1126,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/GEORGIAN_PS.pm
+++ b/lib/Locale/RecodeData/GEORGIAN_PS.pm
@@ -1126,4 +1126,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/GOST_19768_74.pm
+++ b/lib/Locale/RecodeData/GOST_19768_74.pm
@@ -1082,4 +1082,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/GREEK7.pm
+++ b/lib/Locale/RecodeData/GREEK7.pm
@@ -876,4 +876,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/GREEK7_OLD.pm
+++ b/lib/Locale/RecodeData/GREEK7_OLD.pm
@@ -879,4 +879,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/GREEK_CCITT.pm
+++ b/lib/Locale/RecodeData/GREEK_CCITT.pm
@@ -874,4 +874,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/HP_ROMAN8.pm
+++ b/lib/Locale/RecodeData/HP_ROMAN8.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM037.pm
+++ b/lib/Locale/RecodeData/IBM037.pm
@@ -1141,4 +1141,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM038.pm
+++ b/lib/Locale/RecodeData/IBM038.pm
@@ -946,4 +946,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM1004.pm
+++ b/lib/Locale/RecodeData/IBM1004.pm
@@ -1120,4 +1120,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM1026.pm
+++ b/lib/Locale/RecodeData/IBM1026.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM1047.pm
+++ b/lib/Locale/RecodeData/IBM1047.pm
@@ -1139,4 +1139,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM256.pm
+++ b/lib/Locale/RecodeData/IBM256.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM273.pm
+++ b/lib/Locale/RecodeData/IBM273.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM274.pm
+++ b/lib/Locale/RecodeData/IBM274.pm
@@ -946,4 +946,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM275.pm
+++ b/lib/Locale/RecodeData/IBM275.pm
@@ -946,4 +946,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM277.pm
+++ b/lib/Locale/RecodeData/IBM277.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM278.pm
+++ b/lib/Locale/RecodeData/IBM278.pm
@@ -1139,4 +1139,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM280.pm
+++ b/lib/Locale/RecodeData/IBM280.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM281.pm
+++ b/lib/Locale/RecodeData/IBM281.pm
@@ -946,4 +946,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM284.pm
+++ b/lib/Locale/RecodeData/IBM284.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM285.pm
+++ b/lib/Locale/RecodeData/IBM285.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM290.pm
+++ b/lib/Locale/RecodeData/IBM290.pm
@@ -1016,4 +1016,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM297.pm
+++ b/lib/Locale/RecodeData/IBM297.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM420.pm
+++ b/lib/Locale/RecodeData/IBM420.pm
@@ -1119,4 +1119,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM423.pm
+++ b/lib/Locale/RecodeData/IBM423.pm
@@ -1118,4 +1118,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM424.pm
+++ b/lib/Locale/RecodeData/IBM424.pm
@@ -1060,4 +1060,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM437.pm
+++ b/lib/Locale/RecodeData/IBM437.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM500.pm
+++ b/lib/Locale/RecodeData/IBM500.pm
@@ -1141,4 +1141,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM850.pm
+++ b/lib/Locale/RecodeData/IBM850.pm
@@ -1139,4 +1139,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM851.pm
+++ b/lib/Locale/RecodeData/IBM851.pm
@@ -1136,4 +1136,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM852.pm
+++ b/lib/Locale/RecodeData/IBM852.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM855.pm
+++ b/lib/Locale/RecodeData/IBM855.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM857.pm
+++ b/lib/Locale/RecodeData/IBM857.pm
@@ -1132,4 +1132,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM860.pm
+++ b/lib/Locale/RecodeData/IBM860.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM861.pm
+++ b/lib/Locale/RecodeData/IBM861.pm
@@ -1139,4 +1139,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM862.pm
+++ b/lib/Locale/RecodeData/IBM862.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM863.pm
+++ b/lib/Locale/RecodeData/IBM863.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM864.pm
+++ b/lib/Locale/RecodeData/IBM864.pm
@@ -1124,4 +1124,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM865.pm
+++ b/lib/Locale/RecodeData/IBM865.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM866.pm
+++ b/lib/Locale/RecodeData/IBM866.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM868.pm
+++ b/lib/Locale/RecodeData/IBM868.pm
@@ -1088,4 +1088,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM869.pm
+++ b/lib/Locale/RecodeData/IBM869.pm
@@ -1121,4 +1121,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM870.pm
+++ b/lib/Locale/RecodeData/IBM870.pm
@@ -1117,4 +1117,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM871.pm
+++ b/lib/Locale/RecodeData/IBM871.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM874.pm
+++ b/lib/Locale/RecodeData/IBM874.pm
@@ -1074,4 +1074,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM875.pm
+++ b/lib/Locale/RecodeData/IBM875.pm
@@ -1124,4 +1124,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM880.pm
+++ b/lib/Locale/RecodeData/IBM880.pm
@@ -1118,4 +1118,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM891.pm
+++ b/lib/Locale/RecodeData/IBM891.pm
@@ -881,4 +881,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM903.pm
+++ b/lib/Locale/RecodeData/IBM903.pm
@@ -881,4 +881,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM904.pm
+++ b/lib/Locale/RecodeData/IBM904.pm
@@ -888,4 +888,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM905.pm
+++ b/lib/Locale/RecodeData/IBM905.pm
@@ -1120,4 +1120,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IBM918.pm
+++ b/lib/Locale/RecodeData/IBM918.pm
@@ -1086,4 +1086,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/IEC_P27_1.pm
+++ b/lib/Locale/RecodeData/IEC_P27_1.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/INIS.pm
+++ b/lib/Locale/RecodeData/INIS.pm
@@ -855,4 +855,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/INIS_8.pm
+++ b/lib/Locale/RecodeData/INIS_8.pm
@@ -770,4 +770,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/INIS_CYRILLIC.pm
+++ b/lib/Locale/RecodeData/INIS_CYRILLIC.pm
@@ -857,4 +857,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_10367_BOX.pm
+++ b/lib/Locale/RecodeData/ISO_10367_BOX.pm
@@ -1003,4 +1003,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_2033_1983.pm
+++ b/lib/Locale/RecodeData/ISO_2033_1983.pm
@@ -723,4 +723,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_5427.pm
+++ b/lib/Locale/RecodeData/ISO_5427.pm
@@ -882,4 +882,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_5427_EXT.pm
+++ b/lib/Locale/RecodeData/ISO_5427_EXT.pm
@@ -779,4 +779,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_5428.pm
+++ b/lib/Locale/RecodeData/ISO_5428.pm
@@ -840,4 +840,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_1.pm
+++ b/lib/Locale/RecodeData/ISO_8859_1.pm
@@ -605,4 +605,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_10.pm
+++ b/lib/Locale/RecodeData/ISO_8859_10.pm
@@ -1141,4 +1141,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_11.pm
+++ b/lib/Locale/RecodeData/ISO_8859_11.pm
@@ -1061,4 +1061,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_13.pm
+++ b/lib/Locale/RecodeData/ISO_8859_13.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_14.pm
+++ b/lib/Locale/RecodeData/ISO_8859_14.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_15.pm
+++ b/lib/Locale/RecodeData/ISO_8859_15.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_16.pm
+++ b/lib/Locale/RecodeData/ISO_8859_16.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_2.pm
+++ b/lib/Locale/RecodeData/ISO_8859_2.pm
@@ -1141,4 +1141,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_3.pm
+++ b/lib/Locale/RecodeData/ISO_8859_3.pm
@@ -1127,4 +1127,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_4.pm
+++ b/lib/Locale/RecodeData/ISO_8859_4.pm
@@ -1141,4 +1141,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_5.pm
+++ b/lib/Locale/RecodeData/ISO_8859_5.pm
@@ -1140,4 +1140,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_6.pm
+++ b/lib/Locale/RecodeData/ISO_8859_6.pm
@@ -1052,4 +1052,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_7.pm
+++ b/lib/Locale/RecodeData/ISO_8859_7.pm
@@ -1131,4 +1131,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_8.pm
+++ b/lib/Locale/RecodeData/ISO_8859_8.pm
@@ -1068,4 +1068,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/ISO_8859_9.pm
+++ b/lib/Locale/RecodeData/ISO_8859_9.pm
@@ -1141,4 +1141,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/KOI8_R.pm
+++ b/lib/Locale/RecodeData/KOI8_R.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/KOI8_RU.pm
+++ b/lib/Locale/RecodeData/KOI8_RU.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/KOI8_T.pm
+++ b/lib/Locale/RecodeData/KOI8_T.pm
@@ -1098,4 +1098,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/KOI8_U.pm
+++ b/lib/Locale/RecodeData/KOI8_U.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/KOI_8.pm
+++ b/lib/Locale/RecodeData/KOI_8.pm
@@ -1006,4 +1006,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/LATIN_GREEK.pm
+++ b/lib/Locale/RecodeData/LATIN_GREEK.pm
@@ -879,4 +879,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/LATIN_GREEK_1.pm
+++ b/lib/Locale/RecodeData/LATIN_GREEK_1.pm
@@ -881,4 +881,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACARABIC.pm
+++ b/lib/Locale/RecodeData/MACARABIC.pm
@@ -1085,4 +1085,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACCROATIAN.pm
+++ b/lib/Locale/RecodeData/MACCROATIAN.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACCYRILLIC.pm
+++ b/lib/Locale/RecodeData/MACCYRILLIC.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACGREEK.pm
+++ b/lib/Locale/RecodeData/MACGREEK.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACHEBREW.pm
+++ b/lib/Locale/RecodeData/MACHEBREW.pm
@@ -1053,4 +1053,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACICELAND.pm
+++ b/lib/Locale/RecodeData/MACICELAND.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACINTOSH.pm
+++ b/lib/Locale/RecodeData/MACINTOSH.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACROMANIA.pm
+++ b/lib/Locale/RecodeData/MACROMANIA.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACTHAI.pm
+++ b/lib/Locale/RecodeData/MACTHAI.pm
@@ -1125,4 +1125,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACTURKISH.pm
+++ b/lib/Locale/RecodeData/MACTURKISH.pm
@@ -1133,4 +1133,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MACUKRAINE.pm
+++ b/lib/Locale/RecodeData/MACUKRAINE.pm
@@ -1137,4 +1137,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MAC_IS.pm
+++ b/lib/Locale/RecodeData/MAC_IS.pm
@@ -1131,4 +1131,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MAC_SAMI.pm
+++ b/lib/Locale/RecodeData/MAC_SAMI.pm
@@ -1138,4 +1138,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/MAC_UK.pm
+++ b/lib/Locale/RecodeData/MAC_UK.pm
@@ -1135,4 +1135,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/NATS_DANO.pm
+++ b/lib/Locale/RecodeData/NATS_DANO.pm
@@ -881,4 +881,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/NATS_SEFI.pm
+++ b/lib/Locale/RecodeData/NATS_SEFI.pm
@@ -881,4 +881,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/NEXTSTEP.pm
+++ b/lib/Locale/RecodeData/NEXTSTEP.pm
@@ -1133,4 +1133,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/SAMI_WS2.pm
+++ b/lib/Locale/RecodeData/SAMI_WS2.pm
@@ -1128,4 +1128,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/TIS_620.pm
+++ b/lib/Locale/RecodeData/TIS_620.pm
@@ -1059,4 +1059,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/US_ASCII.pm
+++ b/lib/Locale/RecodeData/US_ASCII.pm
@@ -220,4 +220,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/UTF_8.pm
+++ b/lib/Locale/RecodeData/UTF_8.pm
@@ -187,4 +187,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/VISCII.pm
+++ b/lib/Locale/RecodeData/VISCII.pm
@@ -1126,4 +1126,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/RecodeData/_Encode.pm
+++ b/lib/Locale/RecodeData/_Encode.pm
@@ -92,4 +92,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/TextDomain.pm
+++ b/lib/Locale/TextDomain.pm
@@ -916,6 +916,7 @@
 =item B<N__np (MSGCTXT, MSGID, MSGID_PLURAL, COUNT)>
 
 Marks B<MSGID> as N__n() does, but in the context B<MSGCTXT>.
+
 =back
 
 =head1 EXPORTED VARIABLES
@@ -1062,4 +1063,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/Util.pm
+++ b/lib/Locale/Util.pm
@@ -1206,4 +1206,3 @@
 cperl-continued-statement-offset: 2
 tab-width: 4
 End:
-=cut
--- a/lib/Locale/gettext_pp.pm
+++ b/lib/Locale/gettext_pp.pm
@@ -1088,5 +1088,3 @@
 perl-label-offset: -4
 tab-width: 4
 End:
-
-=cut
--- a/lib/Locale/gettext_xs.pod
+++ b/lib/Locale/gettext_xs.pod
@@ -102,5 +102,3 @@
 perl-label-offset: -4
 tab-width: 4
 End:
-
-=cut
